### PR TITLE
Brett Schwartz Capstone Project

### PR DESCRIPTION
**Pull request by bschwartz10**

### CODE REVIEW

File: 1701-b/4module/capstone_projects/project_proposals/brett.md [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
 1-3 sentences describing what type of user your app would be applicable to.
 
+My app will be most applicaable to the 41 million fantasy footballs players in the country. 
+
 ### New Techniques
 
 Which new techniques are you hoping to implement?
 
+React.js, css grids, local storage.
+
```
Comment (neight-allen) [inline 1701-b/4module/capstone_projects/project_proposals/brett.md]: For a true SPA, you'll also need to use react router. You may want to still wrap your application in Rails or Express to serve the different pages. Unless you truly only have one page. Then a single HTML file is fine.

File: 1701-b/4module/capstone_projects/project_proposals/brett.md [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
 1-3 sentences describing the problem that you are trying to solve.
 
+The trading functionality built into current fantasy football platforms is very stagnant. It requires mutliple page loads and you have to reset the entire process if you want to start a new trade. 
+
 ### Solution
 
 1-3 sentences describing how your application will solve that problem.
 
+I want to provide a single page application that will condense all these actions onto a single page without the need for reloading/starting over. 
+
```
Comment (neight-allen) [inline 1701-b/4module/capstone_projects/project_proposals/brett.md]: What are you going to use for your data source? Is there a good free source for player stats?

Comment (bschwartz10) [inline 1701-b/4module/capstone_projects/project_proposals/brett.md]: I downloaded this data as a csv file and imported it into my rails app. (https://www.pro-football-reference.com/years/2015/fantasy.htm)

File: 1701-b/4module/capstone_projects/project_proposals/brett.md (hunk 1 above)
Comment (bschwartz10) [inline 1701-b/4module/capstone_projects/project_proposals/brett.md]: I will do some research into react router. My plan right now is to serve up a rails api and consume it with React on the front end. If you have suggestions/opinions i'd love to discuss it with you.

**Pull request merged**